person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id, :parent_permission
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    is_of_age? || @parent_permission
```

Comment:
- [ ] Kindly make sure to use the correct name for the age function as you defined.

Reply:
@omarsalem7 Thanks for the review, I was getting a linter error for that, but let me change it to its original name.

Reply:
is_of_age? method throws an error as is, but when I follow the suggested refactored name of_age?, No errors are generated
![err](https://user-images.githubusercontent.com/100305514/192277815-26a69361-0caa-4afb-95c6-2b499f9d8c0f.png)